src/Model/Behavior/SequenceBehavior.php

```diff
@@ -259,6 +246,89 @@ public function beforeDelete(Event $event, Entity $entity)
     }
 
     /**
+     * Decrease the position of the entity on the list
+     *
+     * If a "higher" entity exists, this will also swap positions with it
+     *
+     * @param \Cake\Datasource\EntityInterface $entity The entity that is going to be saved.
+     * @return bool
+     */
+    public function moveUp(EntityInterface $entity)
```

Comment:
Yeah I don't think that's quite necessary, and then lends itself to features like "move all the way to top/bottom". Maybe as a separate PR, but this code is probably more useful than the other features.